Classes/Fusion/Helper/WorkspaceHelper.php

```diff
@@ -22,6 +22,7 @@
 
 /**
  * The Workspace helper for EEL contexts
+ * @todo EEL helpers are still to be declared as internal
```

Comment:
And why you did not declare it ;)

Reply:
because this would be to controversy to declare that in this pr. Will be discussed in a followup if the helper was actually useful in some obscure case.